Components/FixedImagePyramids/OpenCLFixedGenericPyramid/elxOpenCLFixedGenericPyramid.hxx

```diff
@@ -202,7 +202,9 @@ OpenCLFixedGenericPyramid< TElastix >
   if( computedUsingOpenCL )
   {
     // Graft output
-    this->GraftOutput( this->m_GPUPyramid->GetOutput() );
+    for( int i=0 ; i < this->GetNumberOfLevels() ; i++ ){
```

Comment:
It seems like `this->GetNumberOfLevels()` could be taken out of the for-loop iteration.

Reply:
This change makes sense as well. @N-Dekker could you implement your suggestion?